src/test/java/org/jboss/aerogear/keycloak/metrics/PrometheusExporterTest.java

```diff
@@ -265,28 +265,28 @@ public void shouldCorrectlyRecordGenericAdminEvents() throws IOException {
 
     @Test
     public void shouldCorrectlyRecordResponseDurations() throws IOException {
-        PrometheusExporter.instance().recordRequestDuration(5, "GET", "admin,admin/serverinfo");
+        PrometheusExporter.instance().recordRequestDuration(200, 5, "GET", "admin,admin/serverinfo", "auth/realm");
```

Comment:
A doubt : should the URI be only "auth/realm" or it can be other paths as well? (since all the tests use this , wanted to understand)

Reply:
the `auth/realm` URI is just an example as part of the test, Any URI path can be used

Reply:
ok got it. thanks!

Reply:
I think it would be good to wait for @pb82. As I understand it just using the RAW URI caused issues in the past especially when there were unique ids in the path as it created too many timeseries. Ideally these URIs would be translated into something like ``` realms/{realm}/protocol/openid-connect/token ``` however that could be a lot of effort so perhaps better to choose some important APIs and just covers those initially.

Reply:
The openid-connect ones would be important ones along with service account mgmt

Reply:
@CathalOConnorRH the raw URL should not be added as a label, this is what caused https://github.com/aerogear/keycloak-metrics-spi/issues/64 in the past

Reply:
@CathalOConnorRH but it looks like you're already using the resource extractor instead of the raw URL. That's better, but keep in mind that every combination of labels leads to a new exported histogram (and that means one additional mertric per bucket plus sum and total). So if we keep adding labels to histograms the amount of metrics will grow (and has grown to huge numbers in the past).